### ISSUE TITLE
Change status page endpoints for RASTER and STAC 

### ISSUE THREAD

**Issue by amarouane-ABDELHAK** [US-GHG-Center/ghg-statuspage#1]
# Description 
Update the [GHGC status page](https://nasa-impact.github.io/ghg-statuspage/) to point to the new STAC and Raster APIs
Raster API: [earth.gov/ghgcenter/api/raster/docs](http://earth.gov/ghgcenter/api/raster/docs)
STAC API: [earth.gov/ghgcenter/api/stac/docs](http://earth.gov/ghgcenter/api/stac/docs)

**Comment by Jeanne-le-Roux** [comment]
@amarouane-ABDELHAK btw I'm currently getting a 404 error for the [status page](https://nasa-impact.github.io/ghg-statuspage/)

**Comment by amarouane-ABDELHAK** [comment]
Yes, we moved the status page to [US-GHG-Center](https://github.com/US-GHG-Center/ghg-statuspage) github org instead of NASA-IMPACT  the working link is https://us-ghg-center.github.io/ghg-statuspage/

**Comment by amarouane-ABDELHAK** [comment]
Endpoints where updated to point to earth.gov
Current endpoint 
stac_api=https://earth.gov/ghgcenter/api/stac/_mgmt/ping
raster_api=https://earth.gov/ghgcenter/api/raster/cog/viewer
mosaic=https://earth.gov/ghgcenter/api/raster/mosaic/list
dashboard=https://earth.gov/ghgcenter
feedback=https://docs.google.com/forms/d/e/1FAIpQLSeVWCrnca08Gt_qoWYjTo6gnj1BEGL4NCUC9VEiQnXA02gzVQ/viewform
links=https://nasa-impact.github.io/link-checker/
